package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Memory leak testing for node. Write tests using your favorite test runner.",
   "main": "lib/index.js",
   "scripts": {
-    "test": "standard lib/**/*.js"
+    "test": "node_modules/.bin/standard lib/**/*.js && node_modules/.bin/mocha test"
```

Comment:
:+1: Sure thing! Here's that section in the docs, in case you're interested in reading more: https://docs.npmjs.com/misc/scripts#path